templates/zerver/api/upload-image.md

```diff
@@ -0,0 +1,40 @@
+# Upload an image
+
+Upload a single image file and get the corresponding URI.
```

Comment:
This should be "Upload a file" -- files don't have to be images (e.g. PDF, etc.)